setup.py

```diff
@@ -8,8 +9,10 @@
     package_dir={'': 'src'},
     packages=['urdf_parser_py', 'urdf_parser_py.xml_reflection'],
     data_files=[
-        ('scripts/display_urdf',
-        ['package.xml']),
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+        ('share/' + package_name + '/scripts', glob('scripts/display_urdf')),
```

Comment:
Does this really install all scripts correctly? I would have thought we should rather fix the original [CMakeLists.txt](https://github.com/ros/urdf_parser_py/blob/melodic-devel/CMakeLists.txt), but I'm fine if this works for now

Reply:
Ok, I see that per migration guide pure python packages should just run the `setup.py`